src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -27,8 +27,8 @@
   <PropertyGroup>
     <ContainerName>build-$([System.Guid]::NewGuid().ToString("N"))</ContainerName>
     <ContainerName>$(ContainerName.ToLower())</ContainerName>
-    <FuncTestListFilename>FuncTests.$(OSGroup).$(Platform)$(ConfigurationGroup).json</FuncTestListFilename>
-    <PerfTestListFilename>PerfTests.$(OSGroup).$(Platform)$(ConfigurationGroup).json</PerfTestListFilename>
+    <FuncTestListFilename>FuncTests.$(BuildConfiguration).json</FuncTestListFilename>
+    <PerfTestListFilename>PerfTests.$(BuildConfiguration).json</PerfTestListFilename>
     <!-- Test builds consist of the tests that are platform specific in one root, plus others in AnyOS. -->
     <AnyOSPlatformConfig>AnyOS.AnyCPU.$(ConfigurationGroup)</AnyOSPlatformConfig>
```

Comment:
At some point we should also fix these so that don't make assumptions like this. In fact as part of our engineering work happening in dev/eng branch this will all be broken.